spec/spec_helper_acceptance.rb

```diff
@@ -4,12 +4,11 @@
 
 $LOAD_PATH << File.join(__dir__, 'acceptance/lib')
 
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
```

Comment:
It was necessary to move this out of the `Rspec.configure` block below.  Otherwise, `facter` isn't available when `if fact('os.release.major') == '8'` runs.